src/setupTests.js

```diff
@@ -0,0 +1,6 @@
+// react-testing-library renders your components to document.body,
```

Comment:
Let's create a new folder to keep all the `test` configurations, mocks, etc... in a single place. Please create a `tests/setup` folder at the root level and place this file in there.

```
+ root
  + scripts
  + src
  + tests
    + setup
    + mocks
    + reports
```